src/backend/core/models.py

```diff
@@ -325,3 +337,47 @@ def clean_fields(self, exclude=None):
         else:
             raise ValidationError({"name": f'Room name "{self.name:s}" is reserved.'})
         super().clean_fields(exclude=exclude)
+
+    def start_recording(self):
+        """Create a new related recording object to which Livekit will be able to save a file."""
+        return Recording.objects.create(room=self)
+
+
+class Recording(BaseModel):
```

Comment:
Should we store the type of recording—audio, video, or both—or rely on LiveKit Egress types for classification?